interface/patient_file/summary/create_portallogin.php

```diff
@@ -165,13 +170,22 @@ function filterTwigTemplateData($twig, $pid, $templateName, $data)
         error_log('OpenEMR Error : OpenEMR is not working because unable to create a hash.');
         die("OpenEMR Error : OpenEMR is not working because unable to create a hash.");
     }
-    array_push($query_parameters, $hash);
 
+    array_push($query_parameters, $hash);
     array_push($query_parameters, $pid);
+
+    EventAuditLogger::instance()->newEvent(
+        "patient access",
```

Comment:
oops, just noted this. for this parameter, would not have spacing since think throws off parsing of it in logger code. would use dash instead `patient-access`